public/javascripts/SVLabel/src/SVLabel/task/TaskContainer.js

```diff
@@ -493,6 +493,17 @@ function TaskContainer (navigationModel, neighborhoodModel, streetViewService, s
         }
     };
 
+    /**
+     * Get street id of current task
+     */
+    function getCurrentTaskStreetEdgeId() {
+        return currentTask ? currentTask.getStreetEdgeId() : null;
+    }
+
+    function getCurrentTaskStreetEdgeIdFunction() {
+        return getCurrentTaskStreetEdgeId;
```

Comment:
I don't think that you should need this function that just wraps the call to the `getCurrentTaskStreetEdgeId` function. I think that the `getCurrentTaskStreetEdgeId` is the solution that we needed, but I don't think that this wrapping function adds anything!

Also you shouldn't need to include the word "function" in the name of a function, that's just a bit redundant!